### DISCUSSION TITLE
AWS Amplify User Testing Opportunity

### DISCUSSION THREAD

**Discussion by emilyahsu** [aws-amplify/learn#344] [General]
Hi everyone! I’m Emily, a UX Designer at AWS Amplify, and I’m conducting user interviews to improve the data modeling experience on Amplify. The interview will take at most 1 hour, and as a “thank you”, we’d love to provide you with $100 in AWS credits for your time. If you are interested in participating, please email me at emilyhsu@amazon.com or DM me on Discord (emilyhsu#6135) and we can schedule a time. Thanks!

**Comment by Melrosee288** [comment]
Tell me why AWS deleted one of my cases from the dashboard on my account and It was regarding the piracy on the Amazon cloud front that is being live streamed alongside child exploitations with more than 20 million views per month and AWS completely ignored all of that and they continue to do so to this day and this is been ongoing to my knowledge since November 2021 & Amazon trust and safety team lied about having ever received a report of that nature when they have multiple times prior to that and they will not respond to my emails when I ask them why they lied
